datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1850,31 +1845,46 @@ mod tests {
     fn test_display_graphviz() -> Result<()> {
         let plan = display_plan()?;
 
+        let expected_graphviz = r###"
+// Begin DataFusion GraphViz Plan,
+// display it online here: https://dreampuf.github.io/GraphvizOnline
+
+digraph {
```

Comment:
I tried it and it looks great 👍 

<img width="1382" alt="Screenshot 2023-07-07 at 4 00 51 AM" src="https://github.com/apache/arrow-datafusion/assets/490673/92ffea6e-8169-4b45-9b75-3d937bef4d0c">